app/migrations/seed.py

```diff
@@ -257,32 +259,32 @@ def update_or_create_menu(name, description, parent_id=None):
     print('------------------------------------------------------------')
     print('Done creating USSD Menus')
 
+    
+    
+    print('Creating Business Categories')
+    print('**********************************************************************')
+    business_categories = [
+        {'name': 'Food', 'icon': 'message'},
+        {'name': 'Water', 'icon': 'message'},
+        {'name': 'Energy', 'icon': 'message'},
+        {'name': 'Education', 'icon': 'message'},
+        {'name': 'Health', 'icon': 'message'},
+        {'name': 'General shop', 'icon': 'message'},
+        {'name': 'Environment', 'icon': 'message'},
+        {'name': 'Transport', 'icon': 'message'},
+        {'name': 'Labour', 'icon': 'message'},
+        {'name': 'Other', 'icon': 'message'},
+    ]
+    for business_category in business_categories:
+        try:
+            usage = TransferUsage(name=business_category['name'], icon=business_category['icon'], 
+            priority=1, default=True)
+            db.session.add(usage)
+            db.session.commit()
+                
+        except IntegrityError as e:
+            print(e)
```

Comment:
@RubenGeo if you haven't made this change yet, i wrote some code for find_or_create in this PR: https://github.com/teamsempo/SempoBlockchain/pull/44/commits/88541f8ab5d6f786db2ed3e5007c56806fd82ece#diff-c92355449c362bbdb8e67b8d67d6a8f8R43

Reply:
Cool it would  make sense to use that function then. How would I go about integrating it in my branch? Just copy paste it? Or will that create conflicts?

Reply:
@enjeyw About the priority, to be honest I don't know what it's used for. What should i change it to?

Reply:
yeah, just copy paste it and i'll fix the conflicts later